docs/UserGuide.md

```diff
@@ -345,22 +321,103 @@ Examples:
 
+**:information_source: Notes:**<br>
 * The search is case-insensitive. e.g `Friday` will match `friday`.
 * The order of the keywords does not matter. e.g. `friday monday` will match `monday friday`.
 * Only the tag names are searched.
-* Only full words will be matched e.g. `PartTime` will not match `PartTimes`.
-* Employees whose tag(s) matching at least one keyword will be listed in the `Employee Contact DeatailL` pane
- (i.e. `OR` search).  e.g. `Friday Monday` will return employees whose tags contain `Friday` or `Monday`.
+* Only full words will be matched. e.g. `Fri` will not match `Friday`.
+* Employees whose tag(s) matching at least one keyword will be listed in the _Employee Directory_ pane
+ (i.e. `OR` search).  e.g. `Friday Monday` will find employees whose tags contain `Friday` or `Monday`.
+
+
+Examples:
+* Let's say you want to find out who are the available to work on `Monday` or `Friday`. In this case, the two key
+ words are `Monday` and `Friday`. You can follow these steps.
+
+Steps to find employees who are available to work on `Monday` or `Friday`:
+1. Type `c-tag-find monday friday` into the _Command Box_.
+2. Press `Enter` to execute.
+
+Outcome:
+1. The _Result Display_ will show the total number of employees who are can work on `Friday` or `Monday`.
+
+2. Employees who are can work on `Friday` or `Monday` will be listed in the _Employee Directory_ pane.
+
+{Screenshot}
+
+#### 3.6 Locating all employees who are available to work today: `c-today`
+
+Finds employees whose tags contain today's day (i.e. `Wednesday`, `Tuesday`, etc).
+
+Format: `c-today`
+
+**:information_source: Note:**<br>
+* In tCheck, if an employee contains a tag for a specific day, it means this employee is available on that day
+. You may set an employee's tags when adding the employee into tCheck (Refer to `c-add` command), or edit the
+tag(s) by using editing employee's information feature (Refer to `c-edit` command).
+* The search is case-insensitive. e.g `Friday` will match `friday`.
+* The application will automatically get today's day based on the current computer's date.
+* Only full words will be marched. e.g `Fri` will not match `Friday`
+
+Examples:
+* Let's say today is `Thursday`. You want to find out who are the available manpower for today. You can follow these
+ steps.
+
+Steps to find today's available manpower:
+1. Type `c-today` into the _Command Box_.
+2. Press `Enter` to execute.
+
+Outcome:
+1. The _Result Display_ will show 3 information:
+    
+    a. Today's day. (e.g. Today is THURSDAY.)
+    
+    b. Total number of employees who are available today.
+    
+    c. The success message.
+2. The available manpower for today will be listed in the _Employee Directory_ pane.
+
+{Screenshot}
+
+#### 3.7 Locating all employees who are available to work tomorrow: `c-tomorrow`
+
+Finds employees whose tags contain tomorrow's day (i.e. `Wednesday`, `Tuesday`, etc).
+
+Format: `c-tomorrow`
+
+**:information_source: Note:**<br>
```

Comment:
I am not sure about this, since we may need to explain such tags at front if we are using them?

Reply:
I've adjusted accordingly based on our discussion.